src/System.Data.DataSetExtensions/pkg/System.Data.DataSetExtensions.pkgproj

```diff
@@ -3,9 +3,12 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
     <ProjectReference Include="..\ref\System.Data.DataSetExtensions.csproj">
-      <SupportedFramework>netcoreapp2.0;net461;$(AllXamarinFrameworks)</SupportedFramework>
+      <SupportedFramework>netcoreapp2.0;net45;$(AllXamarinFrameworks)</SupportedFramework>
     </ProjectReference>
     <ProjectReference Include="..\src\System.Data.DataSetExtensions.csproj" />
+    <InboxOnTargetFramework Include="net45">
```

Comment:
@ericstj do we have to also manually add the placeholder like you did in DirectoryServices? https://github.com/dotnet/corefx/blob/master/src/System.DirectoryServices/pkg/System.DirectoryServices.pkgproj#L13